### PR TITLE
save image and actions for imitation learning

### PR DESCRIPTION
**Pull request by bradyz**
Sample usage
In the top level pystk directory,

```
python3 -m examples.play -t icy_soccer_field --save_dir episode_0
```

This will save pairs of images and actions.

### CODE REVIEW

File: examples/play.py [hunk 1]
```diff
@@ -1,8 +1,18 @@
+from pathlib import Path
+from PIL import Image
 import argparse
 import pystk
 from time import time
+import numpy as np
 from . import gui
 
+
+def action_to_numpy(action):
```
Comment (philkr) [inline examples/play.py]: numpy? It's it a string? To make it a bit easier to read, maybe we can make this a dict and save it as `str(dict)`?
```
def action_dict(action):
    return {k: getattr(action,k) for k in ['acceleration', 'brake', 'steer', 'fire', 'drift']}
```

File: examples/play.py [hunk 2]
```diff
@@ -1,8 +1,18 @@
+from pathlib import Path
+from PIL import Image
 import argparse
 import pystk
 from time import time
+import numpy as np
```
Comment (philkr) [inline examples/play.py]: do you actually need numpy? Or can pil use the memoryview?

File: examples/play.py [hunk 3]
```diff
@@ -54,8 +67,17 @@
...
+
         for ui, d in zip(uis, race.render_data):
             ui.show(d)
+
+        if args.save_dir:
+            image = np.array(race.render_data[0].image)
+            action = action_to_numpy(uis[0].current_action)
+
+            Image.fromarray(image).save(args.save_dir / ('image_%06d.png' % n))
+            Path(str(args.save_dir / ('action_%06d.txt' % n))).write_text(action)
```
Comment (philkr) [inline examples/play.py]: Why `Path(str(Path...))`?
```
(args.save_dir / ('action_%06d.txt' % n)).write_text(action)
```
should work, no?

File: examples/play.py [hunk 4]
```diff
@@ -14,18 +24,22 @@
...
+    parser.add_argument('--save_dir', type=Path, required=False)
     args = parser.parse_args()
 
+    if args.save_dir:
+        args.save_dir.mkdir(parents=True, exist_ok=True)
+
     config = pystk.GraphicsConfig.hd()
-    config.screen_width = 800
-    config.screen_height = 600
+    config.screen_width = 128
```
Comment (philkr) [inline examples/play.py]: Maybe, we can make this an argument instead?

File: examples/play.py (hunk 2 above)
Comment (bradyz) [reply examples/play.py]: PIL didn't work

"memoryview object has no attribute '__array_interface__'

File: examples/play.py (hunk 1 above)
Comment (bradyz) [reply examples/play.py]: i like action_dict better, then read from it with eval()?

**Pull request merged**